buildchain/buildchain/constants.py

```diff
@@ -33,12 +33,16 @@
 REPO_ROOT : Path = ISO_ROOT/'packages'
 # Root of the RedHat repositories on the ISO.
 REPO_RPM_ROOT : Path = REPO_ROOT/'redhat'
+# Root of the Debian repositories on the ISO.
+REPO_DEB_ROOT : Path = REPO_ROOT/'debian'
```

Comment:
So, is it Debian or is it Ubuntu? Those are not the same...

Reply:
Debian, we sort by os-family.